client/reader/search-stream/blank.jsx

```diff
@@ -1,5 +1,27 @@
-import React from 'react';
+const React = require( 'react' );
 
-export default function BlankContent() {
-	return <p />;
-}
+const BlankContent = React.createClass( {
```

Comment:
to get translate, `import { localize } from 'i18n-calypso'` and wrap your function in `localize( )`. You'll be passed a prop named `translate` which you can use to translate things.